back/test/data/classic_location.py

```diff
@@ -23,7 +23,7 @@ def default_location_data():
         "seq": 1,
         "visible": 1,
         "created_by": default_user_data()["id"],
-        "type": LocationType.Location,
+        "type": LocationType.ClassicLocation,
     }
```

Comment:
I don't think this file should be renamed because it contains test data for both classic locations and distro spots.

Reply:
This will also fix the test setup which currently fails: https://app.circleci.com/pipelines/github/boxwise/boxtribute/1753/workflows/aec87eba-db71-44e0-85ae-7d4a8474eece/jobs/7853